app/services/customers/create_service.rb

```diff
@@ -7,6 +7,7 @@ class CreateService < BaseService
     def create_from_api(organization:, params:)
       customer = organization.customers.find_or_initialize_by(external_id: params[:external_id])
       new_customer = customer.new_record?
+      shipping_address = params.key?(:shipping_address) ? params[:shipping_address] : {}
```

Comment:
would you consider this shorter and concise way?

```suggestion
      shipping_address = params[:shipping_address] || {}
```

Reply:
updated